spec/models/volunteer_spec.rb

```diff
@@ -75,11 +75,12 @@
   describe "#made_contact_with_all_cases_in_days?" do
     let(:volunteer) { create(:volunteer) }
     let(:casa_case) { create(:casa_case) }
-    let(:create_case_contact) {
-      ->(occurred_at, contact_made) {
+    let(:create_case_contact) do
+      lambda { |occurred_at, contact_made|
```

Comment:
Not sure this change was necessary...

Reply:
I am happy with lambda over `->` as it is more google-able. it's a little odd to me that `standardrb` doesn't enforce it. As long as they are fully equivalent for test setup (i.e. the tests don't fail or falsely-pass) and the linter likes it, I am happy with the more searchable term.